hive.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: hive
+  version: 4.0.0
+  epoch: 0
+  description: Apache Hive
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-8
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - maven
+      - openjdk-8
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/java-1.8-openjdk
+    HIVE_VERSION: 4.0.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/hive
+      tag: rel/release-${{package.version}}
+      expected-commit: 183f8cb41d3dbed961ffd27999876468ff06690c # rel/release-4.0.0
+
+  - uses: patch
+    with:
+      # This patch covers license check for the goal org.apache.rat:apache-rat-plugin:0.13:check
+      # because the .gitconfig file for safe directories at build time should be ignored.
+      patches: rat.patch
+
+  - runs: |
+      mvn package -Dhadoop.version=3.3.6 -DskipTests -Dmaven.javadoc.skip=true -Pdist
```

Comment:
Hadoop 3.4.0 is not yet supported. Work to support it is in progress [here](https://github.com/apache/hive/pull/5187).